src/azure-cli/azure/cli/command_modules/alertsmanagement/commands.py

```diff
@@ -0,0 +1,49 @@
+from azure.cli.core.commands import CliCommandType
+from azure.cli.command_modules.alertsmanagement._client_factory import (
+    alerts_mgmt_client_factory,
+    smart_groups_mgmt_client_factory,
+    action_rules_mgmt_client_factory)
+from ._exception_handler import alertsmanagement_exception_handler
+
+
+def load_command_table(self, _):
+    alertsmanagement_alert_util = CliCommandType(
+        operations_tmpl='azure.mgmt.alertsmanagement.operations#AlertsOperations.{}',
+        client_factory=alerts_mgmt_client_factory,
+        exception_handler=alertsmanagement_exception_handler
+    )
+
+    alertsmanagement_smart_group_util = CliCommandType(
+        operations_tmpl='azure.mgmt.alertsmanagement.operations#SmartGroupsOperations.{}',
+        client_factory=smart_groups_mgmt_client_factory,
+        exception_handler=alertsmanagement_exception_handler
+    )
+
+    alertsmanagement_action_rule_util = CliCommandType(
+        operations_tmpl='azure.mgmt.alertsmanagement.operations#ActionRulesOperations.{}',
+        client_factory=action_rules_mgmt_client_factory,
+        exception_handler=alertsmanagement_exception_handler
+    )
+
+    with self.command_group('alertsmanagement alert', alertsmanagement_alert_util,
```

Comment:
Hello. Is this module related with monitor? Why don't we include this module under "az monitor"?

Reply:
@MyronFanQiu, yes that should be definitely considered